docs/src/components/ColorSchemes.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+import { createComponent, ComponentSlotStylesInput, ThemePrepared, Grid } from '@stardust-ui/react'
+
+import ColorBox from './ColorBox'
+
+type ColorVariantsProps = {
+  name?: string
+  themes?: ThemePrepared[]
+}
+
+export const colorVariantsStyles: ComponentSlotStylesInput<ColorVariantsProps> = {
+  root: {
+    border: '1px solid transparent',
+    borderRadius: '.25rem',
+    overflow: 'hidden',
+  },
+}
+
+const ColorSchemes = createComponent<ColorVariantsProps>({
+  displayName: 'ColorVariants',
+  render: ({ name, themes, stardust: { classes } }) => {
+    if (themes.length === 0) return <></>
+    const schema = themes[0].siteVariables.colorScheme[name]
+    const elements = []
+
+    Object.keys(schema).forEach(key => {
+      let isFirstElement: boolean = true
+      for (let i = 0; i < themes.length; i++) {
+        if (isFirstElement) {
+          elements.push(<ColorBox name={key} key={`${i}schema`} size="small" value={undefined} />)
+          isFirstElement = false
+        }
+        const value = themes[i].siteVariables.colorScheme[name][key]
+        elements.push(<ColorBox key={`${i}${key}`} size="small" value={value} />)
+      }
+    })
```

Comment:
nit: I tried to rewrite it in more functional way:

```jsx
const colorSchemes = _.map(
  themes,
  theme => theme.siteVariables.colorScheme[name]
);
const elements = _.flatMap(_.head(colorSchemes), (value, token) => [
  <ColorBox
    name={token}
    key={`${token}schema`}
    size="small"
    value={undefined}
  />,
  ..._.map(colorSchemes, (colorScheme, i) => (
    <ColorBox key={`${token}${i}`} size="small" value={colorScheme[token]} />
  ))
]);

```

Reply:
Actually, one more note:

![image](https://user-images.githubusercontent.com/14183168/56355512-29e53800-61df-11e9-8ea3-390956530aaf.png)

We provide way to copy color values, but what about tokens?
What about option for copy: `theme.siteVariables.colorScheme[name]`?

Reply:
The tokens are just human readable (the actual token may be something like `colorScheme.primary.borderDefault`, so not sure if this will bring any value to the user...

Reply:
understand the reasons of @layershifter proposal - however, would also vote for leaving it as is for now. One of the reasons for that is that I wouldn't want to propose the specific API at this point in the copy-paste functionality, as it may change in future. I think that we are providing enough information to the client for being able to consume the color.

At the same time, agree that this might be improved in future